waitress/tests/test_task.py

```diff
@@ -202,6 +202,23 @@ def test_build_response_header_v11_200_no_content_length(self):
         self.assertEqual(inst.close_on_finish, True)
         self.assertTrue(('Connection', 'close') in inst.response_headers)
 
```

Comment:
@tseaver @bertjwregeer Added test for HTTP 1xx (specifically 100 Continue) code. Test is the same as for 204. If you want more in-depth testing, please ask someone with a good knowledge of waitress.